### DISCUSSION TITLE
Help me if you are an AI developer

### DISCUSSION THREAD

**Discussion by popstar7** [Finalgoal231/finalGoal#48] [Study new technical stack] [answered]
Hello, everyone.👋
I am an AI developer and I am going to train a model.
The input is a image and a prompt (represent the other status of that person)
The result is a image based on the prompt and image.
If you have any opinion, please answer to this discussion.

**Comment by noahstoner** [comment, marked as answer]
Hello
Nice to meet you.
I read your problem carefully and I can help you.
Do you know stable-diffusion?
This is image2image model.
You can use Dreambooth for your purpose.
Download this model and test once.
I wish this will help you.

**Reply by popstar7** [reply]
Wonderful, I will try once
